modules/migration-about-configuring-proxies.adoc

```diff
@@ -132,7 +132,26 @@ spec:
     type: Deny
 ----
 
-[id="configuring-supplemental-groups-for-rsync-pods{context}"]
+[id="choosing-alternate-endpoints-for-data-transfer_{context}"]
+=== Choosing alternate endpoints for data transfer
+
+By default, DVM uses an OpenShift Container Platform route as an endpoint to transfer PV data to destination clusters. You can choose another type of supported endpoint, if cluster topologies allow.
+
+For every cluster, you can configure an endpoint by setting an `rsync_endpoint_type` variable on the appropriate *destination* cluster in your `MigrationController` CR:
```

Comment:
s/every/each ?

You can configure this per-cluster, not one configuration for all clusters, right?

Reply:
Fixed; thanks.